### DISCUSSION TITLE
List of records that have implemented "IRequest<>"

### DISCUSSION THREAD

**Discussion by gokmensercan** [LuckyPennySoftware/MediatR#866] [Q&A]
How can I get all the handler name list in the project?

Is there a working method in "MediatR"?

![image](https://user-images.githubusercontent.com/11184522/221949323-fe15169d-f432-4bf0-bae3-531febcd0d92.png)

**Comment by jbogard** [comment]
No, it's not something MediatR will be responsible for. It relies completely on the container to resolve handlers. MediatR can help register them, but it's not at all required. Your best bet is to look at how to enumerate services in your container and find handlers there.
